rooms/views.py

```diff
@@ -223,3 +286,350 @@ def get(self, request, room_id):
+                
+                # User 정보가 host일 경우, 현재 자기가 호스팅하고 RoomReview의 권한을 판단해야함
+                host_auth = 0
+                if user_infos.host and (reviews.first().room.host_id if len(reviews) else 0) == user_infos.id: 
+                    host_auth = 1
+
+                user_info['user_id']    = user_infos.id
+                user_info['user_name']  = user_infos.name
+                user_info['host_auth']  = host_auth
+                user_info['reply_auth'] = 1 if pay_count > review_count else 0
+
+            result = {
+                'room_id'       : room_id,
+                'next_page'     : offset + 1 if len(reviews) else 1,
+                'reviews_count' : RoomReview.objects.all().aggregate(Count('id'))['id__count'],
+                'point'   : [
+                    {
+                        'id'    : evaluation.id,
+                        'name'  : evaluation.name,
+                        'point' : FOMATTER(ReviewEvaluation.objects.filter(
+                                    review_id__in=[review['id'] for review in reviews.values('id')],
+                                    evaluation_id=evaluation.id
+                                ).aggregate(Avg('point'))['point__avg'])
+                    } for evaluation in evaluations
+                ],
+                'comment': [
+                    {
+                        'review_id'     : review.id,
+                        'user_id'       : review.user.id,
+                        'mine_comment'  : 1 if review.user.id == user_id else 0,
+                        'user_name'     : review.user.name,
+                        'profile_image' : review.user.socialflatform_set.filter(user_id=review.user_id).first().profile_image 
+                                          if review.user.socialflatform_set.filter(user_id=review.user_id).exists() else '',
+                        'content'       : review.content if not review.is_deleted else '삭제된 메세지 입니다.',
+                        'depth'         : review.depth,
+                        'group_id'      : review.group_id,
+                        'created_at'    : review.created_at.strftime('%Y-%m-%d')
+                    } for review in reviews
+                ]
+            }
+
+            result['point_average']   = FOMATTER(reduce(lambda sum, dic: sum + dic['point'], result['point'], 0) / len(result['point'] or 0))
+            result['login_user_info'] = user_info
+            
+            return JsonResponse({'reviews' : result}, status=200)
+        except ValueError:
+            return JsonResponse({'MESSAGE' : 'INVALID REQUEST'}, status=400)
+
+    ### review 작성 ###
+    @login_required
+    def post(self, request):
+        try:
+            user_id = request.user
+            
+            datas = json.loads(request.body)
+
+            content  = datas['content']
+            room_id  = int(datas['room_id'])
+            group_id = int(datas['group_id']) if datas.get('group_id') else 0
```

Comment:
`.get()` method를 사용하면, key값이 존재하지 않는 경우 0을 반환할 수 있습니다~

```suggestion
            group_id = int(data.get('group_id', 0))
```